synapse/handlers/device.py

```diff
@@ -27,6 +29,21 @@ class DeviceHandler(BaseHandler):
     def __init__(self, hs):
         super(DeviceHandler, self).__init__(hs)
 
+        self.hs = hs
```

Comment:
I'm not sure I necessarily like that, as its not clear that `is_mine_id` isn't a local function.